### PR TITLE
OL stack gradle support Initial blog draft

### PR DESCRIPTION
**Pull request by mezarin**
Parent issue: #1611

### CODE REVIEW

**Review by scottkurz — DISMISSED** [review]
LGTM... for "Learn More" should we point to the developer-experience Gitter channel?   Or is that already part of the blog framework?

**Comment by mezarin** [comment]
@scottkurz, others blog posts mention the developer-experience Gitter page also, so i added a mention in the Learn More section.

File: posts/2021-08-23-open-liberty-stack-gradle.adoc [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+author_picture: https://avatars3.githubusercontent.com/mezarin
+author_github: https://github.com/mezarin
+seo-title: Cloud-native Development of Gradle Built Applications with the Open Liberty Devfile Stack - OpenLiberty.io
+seo-description: Get true-to-production development of Gradle built applications directly in a Kubernetes cluster
+blog_description: "Get true-to-production development of Gradle built applications directly in a Kubernetes cluster"
+---
+= Cloud-native Development of Gradle Built Applications with the Open Liberty Devfile Stack
+Edward Mezarina <https://github.com/mezarin>
+
+The Open Liberty devfile stack now allows you build and deploy new and existing cloud-native applications using Gradle. The stack leverages  https://github.com/WASdev/ci.gradle[Open Liberty Gradle Plugin] capabilities to manage the Liberty server and Gradle built applications.
```
Comment (scottkurz) [inline posts/2021-08-23-open-liberty-stack-gradle.adoc]: Actually https://github.com/OpenLiberty/ci.gradle  but it redirects.

File: posts/2021-08-23-open-liberty-stack-gradle.adoc [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+== Try it out
+
+To begin, you need three things:
+
+- Kubernetes cluster (i.e. Openshift, Minikube). Be sure to log in. Odo will interact with your kubernetes cluster.
+- Openshift Do. If you have not already installed odo, do so now by following the instructions outlined in the link:https://odo.dev[odo] documentation.
+- An application using Gradle. You can use your own application, or use one of the stack provided applications such as the link:https://github.com/OpenLiberty/application-stack-intro[intro] or link:https://github.com/OpenLiberty/application-stack-starters[starter] application. Instruction on how to use the starter application can be found link:https://github.com/OpenLiberty/application-stack/wiki/Using-the-Default-Starter#getting-started[here]. For this blog, we will use the intro application. This is a basic application that uses JAX-RS and Microprofile technologies.
```
Comment (scottkurz) [inline posts/2021-08-23-open-liberty-stack-gradle.adoc]: Microprofile => MicroProfile

File: posts/2021-08-23-open-liberty-stack-gradle.adoc [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+$ odo url list
+
+Found the following URLs for component my-ol-component
+NAME     STATE      URL                                                                     PORT     SECURE     KIND
+ep1      Pushed     http://ep1-my-ol-component-my-project.my.kube.cluster.ibm.com           9080     false      route
+----
+
+The default endpoint name assigned by the stack's devfile is ep1.
+
+To access the rest endpoint for the application, append `/api/resource` to the URL listed in the previous step (i.e. `http://ep1-my-ol-component-my-project.my.kube.cluster.ibm.com/api/resource`) and access it through a browser or through the command line.
```
Comment (scottkurz) [inline posts/2021-08-23-open-liberty-stack-gradle.adoc]: rest endpoint => REST endpoint

**Review by scottkurz — DISMISSED** [review]

**Review by scottkurz — APPROVED** [review]
LGTM

**Pull request merged**